src/components/formik-radiobutton/FormikRadiobutton.tsx

```diff
@@ -107,9 +127,18 @@ export class FormikRadiobutton extends React.Component<Props> {
     const errors = touched ? (this.props.serverErrors && this.props.serverErrors[name]) || error : null;
 
     const checked = this.props.value == value;
+    let tooltip;
+
+    if(this.props.tooltip){
+      tooltip = (
+        <div>
```

Comment:
Hoeft niet nog eens gewrapped te zitten in een div. Indien TooltipIconStyled block-positioned moet zijn kan je er best een div van maken in het styledcomponent ipv een span.